crates/task-impls/src/network.rs

```diff
@@ -170,9 +170,6 @@ impl<TYPES: NodeType> NetworkMessageTaskState<TYPES> {
                         }
                     },
                 };
-                // TODO (Keyao benchmarking) Update these event variants (similar to the
-                // `TransactionsRecv` event) so we can send one event for a vector of messages.
-                // <https://github.com/EspressoSystems/HotShot/issues/1428>
                 broadcast_event(Arc::new(event), &self.internal_event_stream).await;
```

Comment:
This one appears to be a very old ticket pertaining to events that no longer exist, so I'm not sure what the context is, but my thought is that if it were *really* a huge issue, we'd sort it out. To this point, I have not encountered this.